packages/prisma/seed-app-store.ts

```diff
@@ -281,8 +281,8 @@ export default async function main() {
     await createApp("stripe", "stripepayment", ["payment"], "stripe_payment", {
       client_id: process.env.STRIPE_CLIENT_ID,
       client_secret: process.env.STRIPE_PRIVATE_KEY,
-      payment_fee_fixed: process.env.PAYMENT_FEE_FIXED,
-      payment_fee_percentage: process.env.PAYMENT_FEE_PERCENTAGE,
+      payment_fee_fixed: Number(process.env.PAYMENT_FEE_FIXED),
+      payment_fee_percentage: Number(process.env.PAYMENT_FEE_PERCENTAGE),
```

Comment:
Went to do a functional test of this and realized this was necessary, too.

Now everything is working. :)